### DISCUSSION TITLE
有关dialogue-plato的metric.py

### DISCUSSION THREAD

**Discussion by ruolanyang** [PaddlePaddle/PaddleNLP#385] [Q&A] [answered]
想问一下其中intra-distinct和inter-distinct的区别是？

**Comment by xiemoyuan** [comment, marked as answer]
您好，intra-distinct一般是指某个句子的distinct得分，inter-dist 一般是指多个句子的distinct得分。

**Comment by ZeyuChen** [comment]
这类通用讨论我们转移到Dicussion板块
